src/pages/waiting-room-to-car/cat-c/components/vehicle-checks-modal/vehicle-checks-modal.cat-c.page.ts

```diff
@@ -1,4 +1,4 @@
-import { IonicPage, NavController } from 'ionic-angular';
+import { IonicPage, NavController, NavParams } from 'ionic-angular';
```

Comment:
is there a reason for using nav params rather than the store?

Reply:
It's better for the modal to receive the category through parameters, rather than fetching from store, it's similar to how we pass data to components.

Reply:
We use navparams throughout many modals that we have in the app, so its consistent with the way we create modals.